src/d3a/models/area/__init__.py

```diff
@@ -268,6 +268,10 @@ def _cycle_markets(self, _trigger_event=True, _market_cycle=False, deactivate=Fa
             self.redis_ext_conn.event_market_cycle()
 
     def tick(self):
+        if self.redis_ext_conn is not None and not self.strategy:
+            if self.redis_ext_conn.is_aggregator_controlled:
+                self.redis_ext_conn.aggregator.consume_all_area_commands(
```

Comment:
I would remove the load/pv/storage strategy code that calls consume_all_area_commands and call this method only from here. This has the advantage that there is only one place where the aggregator is consuming the commands, instead of calling the same function in the tick method of all the strategies.